[2022-06-30 21:01:50,987] {processor.py:153} INFO - Started process (PID=74251) to work on /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:01:50,987] {processor.py:641} INFO - Processing file /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-30 21:01:50,988] {logging_mixin.py:115} INFO - [2022-06-30 21:01:50,988] {dagbag.py:507} INFO - Filling up the DagBag from /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:01:51,006] {logging_mixin.py:115} INFO - [2022-06-30 21:01:51,006] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-30 21:01:51,006] {logging_mixin.py:115} INFO - [2022-06-30 21:01:51,006] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-30 21:01:51,007] {processor.py:653} WARNING - No viable dags retrieved from /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:01:51,031] {processor.py:161} INFO - Processing /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2022-06-30 21:15:31,907] {processor.py:153} INFO - Started process (PID=87066) to work on /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:15:31,908] {processor.py:641} INFO - Processing file /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-30 21:15:31,909] {logging_mixin.py:115} INFO - [2022-06-30 21:15:31,909] {dagbag.py:507} INFO - Filling up the DagBag from /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:15:31,926] {logging_mixin.py:115} INFO - [2022-06-30 21:15:31,925] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-30 21:15:31,926] {logging_mixin.py:115} INFO - [2022-06-30 21:15:31,926] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-30 21:15:31,926] {processor.py:653} WARNING - No viable dags retrieved from /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:15:31,952] {processor.py:161} INFO - Processing /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2022-06-30 21:25:00,416] {processor.py:153} INFO - Started process (PID=95972) to work on /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:25:00,417] {processor.py:641} INFO - Processing file /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2022-06-30 21:25:00,417] {logging_mixin.py:115} INFO - [2022-06-30 21:25:00,417] {dagbag.py:507} INFO - Filling up the DagBag from /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:25:00,451] {logging_mixin.py:115} INFO - [2022-06-30 21:25:00,450] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2022-06-30 21:25:00,451] {logging_mixin.py:115} INFO - [2022-06-30 21:25:00,451] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2022-06-30 21:25:00,451] {processor.py:653} WARNING - No viable dags retrieved from /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2022-06-30 21:25:00,489] {processor.py:161} INFO - Processing /Users/danielschneider/opt/anaconda3/lib/python3.8/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.073 seconds
